apps/site/pages/docs/setting-up-a-vtex-account.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: '1. Setting up a VTEX account for FastStore'
+sidebar_label: 'Getting Started'
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+<header>
+
+# Part 1: Setting up a VTEX account for FastStore
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+
+This guide is the first step in developing a **FastStore** website integrated with the **VTEX** Platform. In the following section, you will set up your VTEX environment and install all the necessary apps to start developing with FastStore.
+
+Notice that you must complete the following steps before developing your storefront or configuring your website DNS because needed dependencies, such as [VTEX Intelligent Search](https://help.vtex.com/tracks/vtex-intelligent-search), will be installed and configured throughout this process.
+
+---
+
+## Before you start
+
+Before proceeding any further, make sure you have:
+
+1. Access to a **VTEX** account.
+2. The **VTEX IO CLI** installed on your machine. Please refer to [this guide](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-vtex-io-cli-installation-and-command-reference) for more information.
+
+
+## Step by step
+
+You will use the FastStore plugin for the VTEX IO CLI to install, with a single command, all dependencies needed for developing a FastStore website.
+
+1 . Log in to your VTEX account:
+
+<Callout type="info" emoji="ℹ️">
+   Remember to replace the values between curly brackets according to your scenario.
+</Callout>
+
+   ```
+   vtex login {account}
+   ```
```

Comment:
These whitespaces are causing a break in the positioning of the content.
Removing them solves the problem:

| Before | After |
| --- | --- |
| ![Screenshot 2023-05-29 at 15 52 49](https://github.com/vtex/faststore/assets/15722605/b0e7f5f9-5a1b-40b0-8631-b6537535e065) | ![Screenshot 2023-05-29 at 16 01 55](https://github.com/vtex/faststore/assets/15722605/f872b2cb-be4a-4365-add8-7daad5fda1d6) |

Reply:
Good catch! Thanks, @lucasfp13

Reply:
I love my developers with designer eyes 😍